### PR TITLE
#957 Support of Korean, Chinese and Japanese characters in Standalone

### PR DESCRIPTION
**Pull request by IvanMinin**

### CODE REVIEW

**Comment by IvanMinin** [comment]
Updated NotoSans fonts and tests.

Added CJK regular and bold fonts and tests.

Added lazy loading of fonts.

Removed static variables 'key'.

File: api/wasm/indigo-ketcher/CMakeLists.txt [hunk 1]
```diff
@@ -53,6 +53,23 @@ add_custom_target(${PROJECT_NAME}-test
         DEPENDS ${PROJECT_NAME}
         )
 
+if(RENDER_ENABLE_CJK)
+        add_custom_target(${PROJECT_NAME}-test-cjk
+        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/test_cjk.js ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/
```
Comment (AlexanderSavelyev) [inline api/wasm/indigo-ketcher/CMakeLists.txt:58]: why do we copy tests inside cmake build?

Comment (IvanMinin) [reply api/wasm/indigo-ketcher/CMakeLists.txt:58]: This file is copied to run tests inside "build" folder.
The same way as is it done for no CJK tests.

Comment (AlexanderSavelyev) [reply api/wasm/indigo-ketcher/CMakeLists.txt:58]: can we remote hardcoded tests name from the cmake?

Comment (IvanMinin) [reply api/wasm/indigo-ketcher/CMakeLists.txt]: yes, done

**Review by AlexanderSavelyev — APPROVED** [review]

**Pull request merged**